src/current/_includes/releases/v25.1/v25.1.0-alpha.3.md

```diff
@@ -0,0 +1,76 @@
+
+<h3 id="v25-1-0-alpha-3-sql-language-changes">SQL language changes</h3>
+
+- Users can always see and control (pause/resume/cancel) jobs which they own. [#138178][#138178]
+- Provide different options for `CREATE LOGICALLY REPLICATED TABLE`. [#138244][#138244]
+- `CHANGEFEED`s using named external connections now automatically update their configuration when the connection configuration changes. [#138237][#138237]
+- Added support for `DO` statements embedded within PL/pgSQL routines and other `DO` statements. `DO` statements execute a block of code inline as an anonymous function. Currently, only a PL/pgSQL body is allowed. [#138709][#138709]
+- Added support for `DO` statements in SQL, which allow a PL/pgSQL code block to be executed inline. [#138709][#138709]
+- `DO` statements can now be used as body statements in a SQL routine to execute PL/pgSQL code inline. [#138709][#138709]
+
+<h3 id="v25-1-0-alpha-3-operational-changes">Operational changes</h3>
+
+- If a row level TTL job is scheduled to run and the previous scheduled job for that table is still running, the scheduled run will now be skipped rather than waiting for the previous job to complete. [#138336][#138336]
+- Schema object identifiers (e.g. database names, schema names, table names, and function names) are no longer redacted when logging statements in the `EXEC` or `SQL_SCHEMA` channels. If redaction of these names is required, then the new cluster setting `sql.log.redact_names.enabled` can be set to `true`. The default value of the setting is `false`. [#136897][#136897]
+- Object identifiers such as table names, schema names, function names, and type names are no longer redacted in the `SQL_SCHEMA` log channel. [#136897][#136897]
+- The default value of the cluster setting `admission.l0_file_count_overload_threshold` is changed to `4000`. [#138699][#138699]
+- Introduced a metric, `sql.schema_changer.object_count`, that keeps track of the count of schema objects in the cluster. [#138786][#138786]
+- Renamed the `changefeed.min_highwater_advance` cluster setting to `changefeed.resolved_timestamp.min_update_interval` to more accurately reflect its function. Its description in the automatically generated documentation has also been updated. The previous name remains usable for backwards compatibility. [#138673][#138673]
+
+<h3 id="v25-1-0-alpha-3-db-console-changes">DB Console changes</h3>
+
+- Added a `/debug/pprof/fgprof` endpoint to capture off-CPU stack traces. Use of this endpoint will have a noticeable impact on performance while the endpoint is being triggered. [#138779][#138779]
+
+<h3 id="v25-1-0-alpha-3-bug-fixes">Bug fixes</h3>
+
+- In the v2 **Databases > Table** page, the `CREATE` statement will now show up as expected for tables with custom schema names. [#138294][#138294]
```

Comment:
Does the `>` render correctly? Might have to use the character string here.

```suggestion
- In the v2 **Databases > Table** page, the `CREATE` statement will now show up as expected for tables with custom schema names. [#138294][#138294]
```

Reply:
i have updated this to use `&gt;`